### DISCUSSION TITLE
Filament Login or Livewire related problem

### DISCUSSION THREAD

**Discussion by ramiropiro** [filamentphp/filament#5375] [General]
Hi everyone, I'm trying to solve an issue that show an 404 error when login, regular login or breezy
![image](https://user-images.githubusercontent.com/111913304/210184484-30bbdef1-9d23-4b07-9038-c93fa5bd9f36.png)
I believe is happening with any package but I cannot go further. I'm runnin a prepay hosting with apache, first I had an issue that the config wasnt finding livewere assets, I managed to work editing livewere config and now after clicking login that error appears, any suggestions? Exact same issue here [https://github.com/filamentphp/filament/discussions/5341](url)
![image](https://user-images.githubusercontent.com/111913304/210184556-7305300c-9b82-411e-b887-2bf18b91c9a4.png)

All the answers I found was related to Nginx configuration, but i'm runnin apache

**Comment by ArtMin96** [comment]
There is the same problem locally?

**Reply by ramiropiro** [reply]
I think i'm gettin close to solve it, i'm using laragon and works perfect on my computer, but I was facing the same problem trying to access to my computer from another device, so I had to publish livewire assets and config, after that on livewire config 
![image](https://user-images.githubusercontent.com/111913304/210245162-b38e2480-28d9-44e5-a64c-bad5d5099617.png)
..and is workin perfect, accesing from another device to my computer, 
...but replicating this on my cpanel is not working....livewire seems that is not be able to find packages.

**Comment by ramiropiro** [comment]
Solved, I had to add on my subdomain /public, and everything is working perfect.
